drivers/oled/oled_driver.h

```diff
@@ -150,6 +150,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #    endif
 #endif
 
+typedef struct __attribute__((__packed__)) {
+    uint8_t     *current_element;
+    uint16_t    remaining_element_count;
```

Comment:
```suggestion
    uint8_t  *current_element;
    uint16_t remaining_element_count;
```

Reply:
Thank you @fauxpark for both of these suggestions to better match the project style!